src/Sylius/Component/Promotion/Provider/PreQualifiedPromotionsProviderInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Promotion\Provider;
+
+use Sylius\Component\Promotion\Model\PromotionInterface;
+use Sylius\Component\Promotion\Model\PromotionSubjectInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface PreQualifiedPromotionsProviderInterface
```

Comment:
I should be specified, what does service implementing interface should do. In this case, it should provide **pre-qualified** promotions, not just some promotions or all promotions.